index.js

```diff
@@ -258,6 +258,15 @@ function createMainWindow() {
 }
 
 app.once("browser-window-created", (event, win) => {
+	electron.session.defaultSession.webRequest.onHeadersReceived((details, callback) => {
```

Comment:
There is already some code around CSP https://github.com/th-ch/youtube-music/blob/0e99f96f01803600c5b35de9f4c64169dd1625bd/index.js#L482 we can put additional logic there